capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -0,0 +1,65 @@
+### Cloud Providers
+
+The following table outlines the capabilities of {{ site.data.product.title_short }} that are known to work for various cloud providers.
+
+| Feature                                    | Amazon Web Services (AWS) | Microsoft Azure | Google Cloud Platform (GCP) | Red Hat OpenStack Platform | IBM PowerVC via the OpenStack API     |
+| ------------------------------------------ | ------------------------- | --------------- | --------------------------- | -------------------------- | ------------------------------------- |
+| Relationship Discovery                     | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   |
+| Instance Drift Comparison                  | Yes                       | Yes             | No                          | Yes                        | No                                    |
+| Track Instance Genealogy                   | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   |
+| Capacity & Utilization                     | Yes                       | Yes             | Yes                         | Yes                        | No                                    |
+| Capture Cloud Layer Timelines              | No                        | No              | No                          | Yes                        | Yes                                   |
+| Capture Instance Event Timelines           | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   |
+| Discovery - Provider                       | Yes                       | Yes             | No                          | No                         | No                                    |
+| Disk Addition to Instance                  | Yes                       | Yes             | No                          | Yes                        | No                                    |
+| Key Pairs Inventory                        | Yes                       | No              | Yes                         | Yes                        | Yes                                   |
+| Key Pairs Management                       | Yes                       | Yes             | No                          | Yes                        | Yes                                   |
```

Comment:
I don't think we can do Azure key-pair management without key-pair inventory

Reply:
for Azure - do we support both or neither or just inventory for key pairs?

Reply:
Neither